### DISCUSSION TITLE
How do I hide all tabs when exiting developer console?

### DISCUSSION THREAD

**Discussion by InfernoJoes** [Ark2000/PankuConsole#91] [Q&A] [answered]
Is there a function that I have not looked over, that basically hides all tabs? If not, I just want to know how to do this when I close the developer console.

**Comment by Ark2000** [comment, marked as answer]
do you mean, close developer console will hide all other windows, and when opening the developer console, all other windows will show again? 

If this is the case, I can add an option which enables this behavior in settings as a new feature.

**Reply by InfernoJoes** [reply]
Yes, exactly! But I just realized that the purpose of the windows is to report what happens in the game at runtime, my bad. It would be useful to hide all windows when you're not wanting to use them.

**Comment by Ark2000** [comment]
This feature was implemented in https://github.com/Ark2000/PankuConsole/commit/0a528bd141dec2c9140d907146a4a99a043a3a11, see also https://github.com/Ark2000/PankuConsole/issues/92
